internal/provider/util.go

```diff
@@ -184,11 +184,29 @@ func parseTwoPartID(id string) (string, string, error) {
 	return parts[0], parts[1], nil
 }
 
+func parseTwoPartIDInt(id string) (string, int, error) {
```

Comment:
I'm not sure about this util function here. 
Can you implement something like `resourceGitlabProjectMilestoneParseID` instead?

Reply:
... it's more consistent with the other resources. And I think eventually we may need some more generic concept to parse these ids (with whatever type combinations ...).

Reply:
Okey)